docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -49,6 +49,8 @@ image::provisioning/SNMP_Config.png["SNMP community name configuration page with
 These settings are automatically optimized into `snmp-config.xml`.
 For an example of the resulting XML configuration, see <<deep-dive/provisioning/xml-samples.adoc#SNMP-community-xml, Configure SNMP community names>> in the Deep Dive section.
 
+NOTE: It is possible to use metadata expressions in the fields Security Name, Read Community, Write Community, Auth Passphrase, and Privacy Passphrase to reference credentials stored in the Secure Credentials Vault.
+
```

Comment:
we should also mention this only works with global metadata not with any node, interface etc.

Reply:
done.